docs/content/en/docs/pipeline-stages/init.md

```diff
@@ -41,12 +41,12 @@ If this image is one you want Skaffold to build, you'll need to manually set up
 {{</alert>}}
 
 `skaffold` init also recognizes Maven and Gradle projects, and will auto-suggest the [`jib`]({{<relref "/docs/pipeline-stages/builders#/local#jib-maven-and-gradle">}}) builder.
-Currently `jib` artifact detection is disabled by default, but can be enabled using the flag `--XXenableJibInit`.
+As of v1.27.0, `jib` artifact detection is enabled by default.
```

Comment:
This was sufficiently long ago that we can just remove this, I think.

Reply:
Works for me, I can amend this to remove the line entirely once I've been able to sign my CLA.